styles/globals.scss

```diff
@@ -190,11 +190,11 @@ a {
 
 p {
   font-family: "proxima_nova", sans-serif;
-  font-size: 16px;
+  font-size: 20px;
```

Comment:
do we want `em` units here?

Reply:
does it matter here? since these are global declarations